Jenkinsfile

```diff
@@ -1,3 +1,15 @@
+def githubUsernameToSlackName(github_author) {
```

Comment:
Is this mapping correct for our stash repos?

Reply:
Ill test that out